master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -138,7 +139,7 @@ We'll define some network policy through the Kubernetes API.  Run the following
 a NetworkPolicy which allows traffic to nginx pods from any pods in the advanced-policy-demo Namespace. 
 
 ```shell
-kubectl create -f - <<EOF 
+cat << EOF | kubectl create -f -
```

Comment:
yeah I also like it better, didn't know we could do it that way. I think we should also change all the calicoctl ones to use this (in a separate PR) to be consistent. I can do that